src/schema/schema_drop.c

```diff
@@ -118,9 +118,7 @@ __drop_table(
      * Open the table so we can drop its column groups and indexes.
      *
      * Ideally we would keep the table locked exclusive across the drop, but for now we rely on the
```

Comment:
This comment is the one I'd like a tech debt ticket on with a FIXME.

Reply:
Sorry I forgot to link that. Here it is:
[WT-13812 Investigate table locking during session->drop](https://jira.mongodb.org/browse/WT-13812)

Reply:
Added https://github.com/wiredtiger/wiredtiger/pull/11273/commits/c3e6b8c4a3b94fb6ce1c4cea08a3a3195a99e8bd